src/main/java/duke/main/FileManipulation.java

```diff
@@ -0,0 +1,84 @@
+package duke.main;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
+import java.time.LocalDate;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class FileManipulation {
```

Comment:
I noticed all the public classes do not really have javadocs, it might be useful for some of the classes you've introduced yourself